packages/@stylexjs/babel-plugin/__tests__/transform-stylex-createTheme-test.js

```diff
@@ -67,13 +69,21 @@ function createFixture(fixture, { varsFilename, varsLiterals } = {}) {
       ${createKey('radius')}: 10
     });
   `,
-    options,
-  ).code;
+    fixtureTransformOptions,
+  );
 
-  return `
-    ${defineVarsOutput}
+  // Generate the final transform
+  const { code, metadata } = transform(
+    `
+    ${_code}
     ${fixture}
-  `;
+  `,
+    pluginOptions,
+  );
+
+  _metadata.stylex.push(...metadata.stylex);
+
+  return { code, metadata: _metadata };
```

Comment:
This test abstraction exists because we're trying to test the integration between defineVars and createTheme. But, we were only taking snapshots of partial metadata, missing the "default theme" CSS rules. To fix that, I chose to refactor this helper